patterns/footer-social.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Title: Centered Footer with social links
+ * Slug: twentytwentyfive/footer-social
+ * Categories: footer
+ * Block Types: core/template-part/footer
+ * Description: Footer with centered site title and social links
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|50","left":"var:preset|spacing|50","right":"var:preset|spacing|50"}}},"layout":{"type":"constrained"}} -->
```

Comment:
The bottom spacing in Figma is 73px, but I was not sure if it was intentional, so I kept it at spacing preset 50.